packages/fxa-auth-server/package.json

```diff
@@ -21,7 +21,11 @@
     "restart": "pm2 restart pm2.config.js",
     "delete": "pm2 delete pm2.config.js",
     "test": "yarn merge-ftl:test && yarn emails-scss && VERIFIER_VERSION=0 scripts/test-local.sh",
+    "test:unit": "yarn merge-ftl:test && yarn emails-scss && VERIFIER_VERSION=0 RUN_UNIT=1 scripts/test-local.sh",
```

Comment:
What's the `RUN_UNIT` for?

Reply:
Thanks for calling this out. I think I lost part of my commit for auth-server 😢. I got in some conflicts while rebasing and must have clobbered some of the auth server changes accidentally. 

I've restored my changes now. These flags, which I just changed slightly, turn on unit / integration testing in the scripts/test-local.sh. Using an env to do this was an effort to avoid breaking change on that script, since it currently uses glob directly to target test/local, test/remote, etc... Relying on env for these args preserves the the glob logic and was already a pattern in place.